src/exceptions/HO100300.ts

```diff
@@ -1,18 +1,18 @@
 import convertAsnToOrganisationUnit from "src/lib/convertAsnToOrganisationUnit"
+import findException from "src/lib/findException"
 import type { OrganisationUnitCodes } from "src/types/AnnotatedHearingOutcome"
 import type Exception from "src/types/Exception"
+import type { ExceptionPath } from "src/types/Exception"
 import { ExceptionCode } from "src/types/ExceptionCode"
 import type { ExceptionGenerator } from "src/types/ExceptionGenerator"
 import { lookupOrganisationUnitByCode } from "src/use-cases/dataLookup"
 import { asnPath } from "src/use-cases/enrichHearingOutcome/enrichFunctions/enrichCourtCases/errorPaths"
 import populateOrganisationUnitFields from "src/use-cases/populateOrganisationUnitFields"
 
-type FieldPath = (string | number)[]
-
-const COURT_HEARING_LOCATION_PATH: FieldPath =
+const COURT_HEARING_LOCATION_PATH: ExceptionPath =
   "AnnotatedHearingOutcome.HearingOutcome.Hearing.CourtHearingLocation.OrganisationUnitCode".split(".")
 
-const getResultPath = (offenceIndex: number, resultIndex: number): FieldPath => [
+const getResultPath = (offenceIndex: number, resultIndex: number): ExceptionPath => [
```

Comment:
Wonder if we could put all of the error paths in one place and move `errorPaths.ts` to somewhere more generic?

Reply:
Makes sense. I put them all in `lib/errorPaths.ts`